pkg/logs/scheduler/scheduler.go

```diff
@@ -172,6 +174,15 @@ func (s *Scheduler) toSources(config integration.Config) ([]*logsConfig.LogSourc
 	}
 
 	var service *service.Service
+
+	commonGlobalOptions := integration.CommonGlobalConfig{}
+	err = yaml.Unmarshal(config.InitConfig, &commonGlobalOptions)
+	if err != nil {
+		return nil, fmt.Errorf("invalid init_config section for source %s: %s", config.Name, err)
+	}
```

Comment:
Is the `init_config` part mandatory in a check configuration? If not, should we take care of this in here?

Reply:
I don't know if it's required anymore actually

Reply:
`init_config` is not required anymore

Reply:
how will this behave with config from docker labels or kubernetes annotations ?

Reply:
I believe @clamoriniere has a related goal that will encompass this, targeted for 7.19